x-pack/plugins/uptime/server/legacy_uptime/lib/alerts/duration_anomaly.ts

```diff
@@ -226,4 +228,6 @@ export const durationAnomalyAlertFactory: UptimeAlertTypeFactory<ActionGroupIds>
     return { state: updateState(state, foundAnomalies) };
   },
   alerts: UptimeRuleTypeAlertDefinition,
+  getViewInAppRelativeUrl: ({ rule }: GetViewInAppRelativeUrlFnOpts<{}>) =>
```

Comment:
can you please also implement this for Synthetics Status and TLS rule? 
https://github.com/elastic/kibana/blob/main/x-pack/plugins/synthetics/server/alert_rules/status_rule/monitor_status_rule.ts#L38

Reply:
Sure, done!